tutorials/MaxSubstructualAlignment.ipynb

```diff
@@ -0,0 +1,296 @@
+      "\u001b[0;32m/shared/alejandro/miniconda3/lib/python3.6/site-packages/numba/dispatcher.py\u001b[0m in \u001b[0;36m_explain_matching_error\u001b[0;34m(self, *args, **kws)\u001b[0m\n\u001b[1;32m    427\u001b[0m         msg = (\"No matching definition for argument type(s) %s\"\n\u001b[1;32m    428\u001b[0m                % ', '.join(map(str, args)))\n\u001b[0;32m--> 429\u001b[0;31m         \u001b[0;32mraise\u001b[0m \u001b[0mTypeError\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmsg\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    430\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    431\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_search_new_conversions\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m*\u001b[0m\u001b[0margs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkws\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
+      "\u001b[0;31mTypeError\u001b[0m: No matching definition for argument type(s) array(float32, 3d, C), array(float32, 3d, C), reflected list(int64), reflected list(int64), array(int64, 1d, C), int64"
+     ]
+    }
+   ],
+   "source": [
+    "from htmd.ui import *\n",
+    "from htmd.molecule.graphalignment import maximalSubstructureAlignment\n",
+    "config(viewer='ngl')\n",
+    "\n",
+    "#load the molecule which will be used as reference: PDB 2Y02 with resname WHJ\n",
```

Comment:
PEP8 style. Put a space after the # and the comments themselves :) Capitalization is up to you I guess as you like